README.md

```diff
@@ -30,6 +30,33 @@ Additionally, the example code has been released to the public domain (see the
 Quick Start
 ===========
 
+### Windows
+
+You can use vcpkg install libgit2:
```

Comment:
This should probably be "You can use vcpkg _to_ install libgit2:", so there's a "to" missing. Are additional instructions required to include the installed package in your MSVC project then or is this already covered somewhere else?